packages/ui/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@taiko/ui",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "description": "Design Guidelines and Components for Taiko",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "Ethereum",
+    "ZKP",
+    "zkRollup",
+    "Layer2",
+    "Tailwindcss",
+    "Design",
+    "Guidelines"
+  ],
```

Comment:
```suggestion
```
reason for removing keywords: it is a private package so we don't need any SEO keywords here
reason for removing description: duplication of README.md

also would we need a README.md if we decide to use this package instead of `/branding`